### PR TITLE
Fill metadata into Reference as expand Output only consumes its own P…

### PR DESCRIPTION
**Pull request by vicancy**
…ageViewModel

### CODE REVIEW

**Comment by fenxu** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs [hunk 1]
```diff
@@ -113,6 +113,22 @@ private void FillCore(PageViewModel model, IHostService host, string file)
 
         private void FillContent(ReferenceViewModel r, dynamic item)
         {
+            if (item.Metadata != null)
+            {
+                foreach (var pair in item.Metadata)
+                {
+                    switch (pair.Key)
+                    {
+
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs]: minor: remove empty line

Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.ManagedReference/FillReferenceInformation.cs]: remove blank line.

**Comment by superyyrrzz** [comment]
:shipit:

**Pull request merged**